# This file is part of HiGFXback

# requires
REQUIRES="make-build stb-build"

pkg-config --exists --print-errors $REQUIRES || exit 1

# configure
rm -f stb.h
rm -f stb_truetype.h

# build
make

# install
install -d $DESTDIR/bin
install stb_makefont $DESTDIR/bin

install -d $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_6_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_7_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_8_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_9_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_10_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_11_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_12_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_13_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_14_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_15_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_16_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_17_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_18_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_19_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_20_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_21_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_22_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_23_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_24_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_25_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_26_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_27_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_28_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_29_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_30_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_31_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_32_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_33_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_34_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_35_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_36_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_37_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_38_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_39_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_40_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_41_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_42_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_43_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_44_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_45_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_46_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_47_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_48_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_49_latin1.inl $DESTDIR/include/stb_font/latin1/arial
install -m 644 latin1/arial/stb_font_arial_50_latin1.inl $DESTDIR/include/stb_font/latin1/arial

install -d $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_6_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_7_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_8_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_9_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_10_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_11_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_12_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_13_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_14_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_15_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_16_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_17_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_18_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_19_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_20_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_21_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_22_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_23_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_24_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_25_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_26_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_27_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_28_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_29_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_30_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_31_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_32_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_33_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_34_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_35_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_36_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_37_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_38_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_39_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_40_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_41_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_42_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_43_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_44_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_45_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_46_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_47_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_48_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_49_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold
install -m 644 latin1/arial_bold/stb_font_arial_bold_50_latin1.inl $DESTDIR/include/stb_font/latin1/arial_bold

install -d $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_6_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_7_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_8_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_9_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_10_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_11_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_12_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_13_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_14_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_15_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_16_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_17_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_18_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_19_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_20_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_21_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_22_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_23_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_24_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_25_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_26_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_27_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_28_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_29_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_30_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_31_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_32_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_33_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_34_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_35_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_36_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_37_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_38_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_39_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_40_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_41_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_42_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_43_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_44_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_45_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_46_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_47_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_48_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_49_latin1.inl $DESTDIR/include/stb_font/latin1/consolas
install -m 644 latin1/consolas/stb_font_consolas_50_latin1.inl $DESTDIR/include/stb_font/latin1/consolas

install -d $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_6_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_7_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_8_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_9_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_10_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_11_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_12_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_13_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_14_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_15_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_16_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_17_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_18_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_19_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_20_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_21_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_22_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_23_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_24_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_25_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_26_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_27_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_28_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_29_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_30_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_31_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_32_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_33_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_34_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_35_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_36_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_37_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_38_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_39_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_40_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_41_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_42_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_43_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_44_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_45_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_46_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_47_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_48_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_49_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold
install -m 644 latin1/consolas_bold/stb_font_consolas_bold_50_latin1.inl $DESTDIR/include/stb_font/latin1/consolas_bold

install -d $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_6_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_7_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_8_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_9_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_10_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_11_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_12_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_13_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_14_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_15_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_16_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_17_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_18_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_19_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_20_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_21_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_22_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_23_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_24_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_25_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_26_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_27_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_28_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_29_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_30_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_31_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_32_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_33_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_34_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_35_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_36_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_37_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_38_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_39_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_40_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_41_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_42_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_43_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_44_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_45_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_46_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_47_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_48_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_49_latin1.inl $DESTDIR/include/stb_font/latin1/courier
install -m 644 latin1/courier/stb_font_courier_50_latin1.inl $DESTDIR/include/stb_font/latin1/courier

install -d $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_6_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_7_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_8_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_9_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_10_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_11_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_12_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_13_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_14_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_15_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_16_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_17_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_18_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_19_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_20_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_21_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_22_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_23_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_24_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_25_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_26_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_27_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_28_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_29_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_30_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_31_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_32_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_33_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_34_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_35_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_36_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_37_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_38_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_39_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_40_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_41_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_42_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_43_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_44_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_45_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_46_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_47_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_48_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_49_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold
install -m 644 latin1/courier_bold/stb_font_courier_bold_50_latin1.inl $DESTDIR/include/stb_font/latin1/courier_bold

install -d $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_6_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_7_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_8_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_9_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_10_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_11_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_12_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_13_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_14_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_15_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_16_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_17_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_18_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_19_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_20_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_21_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_22_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_23_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_24_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_25_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_26_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_27_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_28_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_29_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_30_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_31_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_32_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_33_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_34_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_35_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_36_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_37_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_38_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_39_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_40_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_41_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_42_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_43_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_44_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_45_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_46_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_47_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_48_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_49_latin1.inl $DESTDIR/include/stb_font/latin1/times
install -m 644 latin1/times/stb_font_times_50_latin1.inl $DESTDIR/include/stb_font/latin1/times

install -d $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_6_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_7_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_8_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_9_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_10_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_11_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_12_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_13_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_14_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_15_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_16_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_17_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_18_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_19_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_20_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_21_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_22_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_23_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_24_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_25_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_26_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_27_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_28_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_29_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_30_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_31_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_32_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_33_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_34_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_35_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_36_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_37_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_38_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_39_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_40_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_41_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_42_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_43_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_44_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_45_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_46_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_47_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_48_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_49_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold
install -m 644 latin1/times_bold/stb_font_times_bold_50_latin1.inl $DESTDIR/include/stb_font/latin1/times_bold

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/stb_font-build.pc << EOF
Name: stb_font
Version: 0.3
Description: font glyphs generated in texture data
Requires: $REQUIRES

exec=\\
/bin/stb_makefont \\
/include/stb_font/latin1/arial/stb_font_arial_6_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_7_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_8_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_9_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_10_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_11_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_12_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_13_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_14_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_15_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_16_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_17_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_18_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_19_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_20_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_21_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_22_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_23_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_24_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_25_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_26_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_27_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_28_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_29_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_30_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_31_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_32_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_33_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_34_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_35_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_36_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_37_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_38_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_39_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_40_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_41_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_42_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_43_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_44_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_45_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_46_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_47_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_48_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_49_latin1.inl \\
/include/stb_font/latin1/arial/stb_font_arial_50_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_6_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_7_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_8_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_9_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_10_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_11_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_12_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_13_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_14_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_15_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_16_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_17_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_18_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_19_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_20_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_21_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_22_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_23_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_24_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_25_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_26_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_27_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_28_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_29_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_30_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_31_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_32_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_33_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_34_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_35_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_36_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_37_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_38_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_39_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_40_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_41_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_42_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_43_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_44_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_45_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_46_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_47_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_48_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_49_latin1.inl \\
/include/stb_font/latin1/arial_bold/stb_font_arial_bold_50_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_6_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_7_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_8_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_9_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_10_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_11_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_12_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_13_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_14_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_15_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_16_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_17_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_18_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_19_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_20_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_21_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_22_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_23_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_24_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_25_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_26_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_27_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_28_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_29_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_30_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_31_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_32_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_33_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_34_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_35_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_36_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_37_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_38_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_39_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_40_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_41_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_42_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_43_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_44_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_45_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_46_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_47_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_48_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_49_latin1.inl \\
/include/stb_font/latin1/consolas/stb_font_consolas_50_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_6_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_7_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_8_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_9_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_10_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_11_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_12_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_13_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_14_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_15_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_16_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_17_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_18_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_19_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_20_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_21_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_22_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_23_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_24_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_25_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_26_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_27_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_28_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_29_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_30_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_31_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_32_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_33_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_34_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_35_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_36_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_37_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_38_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_39_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_40_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_41_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_42_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_43_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_44_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_45_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_46_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_47_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_48_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_49_latin1.inl \\
/include/stb_font/latin1/consolas_bold/stb_font_consolas_bold_50_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_6_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_7_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_8_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_9_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_10_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_11_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_12_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_13_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_14_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_15_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_16_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_17_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_18_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_19_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_20_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_21_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_22_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_23_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_24_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_25_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_26_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_27_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_28_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_29_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_30_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_31_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_32_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_33_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_34_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_35_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_36_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_37_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_38_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_39_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_40_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_41_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_42_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_43_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_44_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_45_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_46_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_47_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_48_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_49_latin1.inl \\
/include/stb_font/latin1/courier/stb_font_courier_50_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_6_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_7_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_8_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_9_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_10_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_11_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_12_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_13_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_14_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_15_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_16_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_17_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_18_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_19_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_20_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_21_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_22_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_23_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_24_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_25_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_26_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_27_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_28_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_29_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_30_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_31_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_32_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_33_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_34_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_35_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_36_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_37_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_38_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_39_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_40_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_41_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_42_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_43_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_44_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_45_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_46_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_47_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_48_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_49_latin1.inl \\
/include/stb_font/latin1/courier_bold/stb_font_courier_bold_50_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_6_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_7_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_8_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_9_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_10_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_11_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_12_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_13_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_14_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_15_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_16_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_17_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_18_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_19_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_20_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_21_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_22_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_23_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_24_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_25_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_26_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_27_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_28_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_29_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_30_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_31_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_32_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_33_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_34_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_35_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_36_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_37_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_38_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_39_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_40_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_41_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_42_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_43_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_44_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_45_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_46_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_47_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_48_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_49_latin1.inl \\
/include/stb_font/latin1/times/stb_font_times_50_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_6_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_7_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_8_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_9_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_10_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_11_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_12_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_13_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_14_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_15_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_16_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_17_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_18_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_19_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_20_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_21_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_22_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_23_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_24_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_25_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_26_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_27_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_28_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_29_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_30_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_31_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_32_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_33_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_34_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_35_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_36_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_37_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_38_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_39_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_40_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_41_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_42_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_43_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_44_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_45_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_46_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_47_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_48_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_49_latin1.inl \\
/include/stb_font/latin1/times_bold/stb_font_times_bold_50_latin1.inl
EOF

exit

# patch
--- stb_font-0.3.orig/Makefile
+++ stb_font-0.3/Makefile
@@ -0,0 +1,9 @@
+CFLAGS += -g -O2
+
+all: stb_makefont
+
+stb_makefont: stb_font_inl_generator.c
+	$(CC) $^ -o $@ $(CFLAGS) -I`$(CC) -print-sysroot`/include/stb -lm
+
+clean:
+	$(RM) stb_makefont

# source
https://nothings.org/stb/font/stb_font_latin1.zip
https://nothings.org/stb/font/stb_font_latin_ext.zip
https://nothings.org/stb/font/stb_font_usascii.zip
https://nothings.org/stb/font/stb_font.zip
